packages/common/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "git+https://github.com/UMAprotocol/protocol.git"
   },
   "scripts": {
-    "test": "echo \"No tests configured\""
+    "test": "mocha mocha-test"
```

Comment:
what does `mocha-test` do?

Reply:
its the folder where teh test is located